include/versions.h

```diff
@@ -0,0 +1,15 @@
+#ifndef VERSIONS_H
+#define VERSIONS_H
+
+// OOT versions in build order
+#define OOT_NTSC_1_2 1
+#define OOT_GC_JP 2
+#define OOT_GC_JP_MQ 3
+#define OOT_GC_US 4
```

Comment:
any issues with using an enum so we dont need to manually specify values?

Reply:
we need the C preprocessor to be able to expand these, which makes me think it won't work as an enum but I haven't tried it

Reply:
I thought enums were processed by cpp, but after looking it up apparently im wrong